server/src/packages/landuse-form/landuse-actions/landuse-actions.controller.ts

```diff
@@ -28,15 +28,24 @@ export class LanduseActionsController {
 
   @Patch('/:id')
   async update(@Body() body, @Param('id') id) {
-    const allowedAttrs = pick(body, LANDUSE_ACTION_ATTRS);
+    let allowedAttrs = pick(body, LANDUSE_ACTION_ATTRS);
     const dateOfPreviousApproval = this.pickDate(body.dcp_dateofpreviousapproval);
     const lapseDateOfPreviousApproval = this.pickDate(body.dcp_lapsedateofpreviousapproval);
     const recordationDate = this.pickDate(body.dcp_recordationdate);
 
+    // unlike other attributes, chosen_zoning_resolution_id
+    // is not always included in the body. So we make an
+    // explicit check for "null".
+    // TODO: Investigate why
+    if (body.chosen_zoning_resolution_id) {
+      allowedAttrs['dcp_zoningresolutionsectionactionispursuantto@odata.bind'] = `/dcp_zoningresolutions(${body.chosen_zoning_resolution_id})`;
```

Comment:
ooh interesting way of doing this setting it on `allowedAttrs`, I like it! The `crmService.update` feels a lot cleaner this way actually